IAMOTZ/ievents
# #152797323 Implement the add-event page with react

#### What does this PR do?
It uses react and redux to implement the add-event page consuming the API endpoint for adding-event
#### Description of Task to be completed?
User should be able to add a new event
#### How should this be manually tested?
- clone the repo
- cd into the project directory
- Run `npm install` to install all the dependency
- Run `npm run react:dev`
- visit the port where the webpack-dev-server started, the signup page should be loaded there
- Fill the form and click register
- The add-event page should show
#### Any background context you want to provide?
- The frontend still uses the locally hosted server, so you would have to create a .env file to fulfill all the needed environment variables, the run `npm run server:dev`
#### What are the relevant pivotal tracker stories?
#152797323
#### Screenshots (if appropriate)
![screencapture-localhost-8080-addevent-1512125891925](https://user-images.githubusercontent.com/26048536/33479921-35391bd6-d68f-11e7-8899-328830eba7f0.png)
#### Questions:
None
